### PR TITLE
HOPSWORKS-295

### PR DESCRIPTION
**Pull request by augustbonds**
Done.

### CODE REVIEW

File: hopsworks-api/src/main/java/io/hops/hopsworks/apiV2/currentUser/UserResource.java [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+      if (pt.getProjectTeamPK().getTeamMember().compareToIgnoreCase(email) == 0) {
+        userDTO.setRole(pt.getTeamRole());
+      }
+    }
+
+    return Util.jsonOk(userDTO);
+  }
+  
+  @Path("/messages")
+  @RolesAllowed({"HOPS_ADMIN", "HOPS_USER"})
```
Comment (tkakantousis) [inline hopsworks-api/src/main/java/io/hops/hopsworks/apiV2/currentUser/UserResource.java]: @augustbonds is this redundant?

File: hopsworks-api/src/main/java/io/hops/hopsworks/apiV2/filter/ApiV2FilterBase.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hops.hopsworks.apiV2.filter;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import java.io.IOException;
+
+public abstract class ApiV2FilterBase implements ContainerRequestFilter{
```
Comment (tkakantousis) [inline hopsworks-api/src/main/java/io/hops/hopsworks/apiV2/filter/ApiV2FilterBase.java]: @augustbonds is this used?

File: hopsworks-api/src/main/java/io/hops/hopsworks/apiV2/projects/PathValidatorV2.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import io.hops.hopsworks.common.exception.AppException;
+import org.apache.hadoop.fs.Path;
+
+import javax.ejb.EJB;
+import javax.ejb.Stateless;
+import javax.ws.rs.core.Response;
+import java.util.logging.Logger;
+
+@Stateless
+public class PathValidatorV2 {
```
Comment (tkakantousis) [inline hopsworks-api/src/main/java/io/hops/hopsworks/apiV2/projects/PathValidatorV2.java]: @augustbonds Merge with PathValidator

**Comment by augustbonds** [comment]
Resubmitting

**Pull request closed without merging**